src/github.com/couchbase/sync_gateway/rest/handler.go

```diff
@@ -501,45 +501,55 @@ func (h *handler) writeStatus(status int, message string) {
 
 // Returns the integer value of a URL query, restricted to a min and max value,
 // but returning 0 if missing or unparseable
-func getRestrictedIntQuery(values url.Values, query string, defaultValue, minValue, maxValue uint64) uint64 {
+func getRestrictedIntQuery(values url.Values, query string, defaultValue, minValue, maxValue uint64, allowZero bool) uint64 {
```

Comment:
Can you add comments to the doc about the allowZero parameter -- maybe something like:

> if allowZero is true, let the value be set to zero rather than forcing it to the minimum value